python3/unittest/test_usb_scan.py

```diff
@@ -88,18 +82,17 @@ def list_devices(self, **kwargs):
                 return MocEnumerator(self.devices)
             elif dev_type == "usb_interface":
                 return MocEnumerator(self.interfaces)
-        return MocEnumerator([])
```

Comment:
This change is for improving code coverage in test_usb_scan.py

Reply:
So this is removed because the case is not hit during testing?

Reply:
yes

Reply:
Then it makes sense to remove the line
```python
         if kwargs.pop("subsystem") == "usb":
```
and replace it with
```python
         assert kwargs.pop("subsystem") == "usb";
```

That way it's obvious when the current assumptions change instead of risking the tests to pass due to an accidental behaviour.